templates/slot/components/slotfinder.html

```diff
@@ -0,0 +1,111 @@
+{{ define "slot_finder" }}
+  <div class="m-2 d-flex flex-wrap">
+    <div class="d-flex m-2">
+      <div class="d-flex flex-column">
+        <label>Slot</label>
+        <div class="d-flex align-items-center">
+          <input id="slotNumber" class="form-control col-6" type="number" value="1" oninput="slotChanged()" />
+          <a id="slotLink"><i class="fas fa-cube mx-2"></i></a>
+        </div>
+      </div>
+      <div class="d-flex flex-column ml-2">
+        <label>Epoch</label>
+        <a id="epochLink" class="d-flex align-items-center">
+          <labal id="epoch">1</labal>
+          <i class="fas fa-history mx-2"></i>
+        </a>
+      </div>
+    </div>
+    <div class="d-flex flex-column m-2">
+      <label>Beaconchain Day</label>
+      <labal id="beaconchainDay">1</labal>
+    </div>
+    <div class="d-flex flex-column m-2">
+      <label>UTC</label>
+      <input id="datepickerUtc" type="text" class="form-control" />
+    </div>
+    <div class="d-flex flex-column m-2">
+      <label>Local</label>
+      <input id="datepickerLocal" type="text" class="form-control" />
+    </div>
+  </div>
+  <small class="d-flex justify-content-end align-items-center m-2">Inpired by <a href="https://slots.symphonious.net" class="ml-2">https://slots.symphonious.net</a></small>
+  <script>
+    var genesisTimeStamp = {{ .ChainGenesisTimestamp }}
+    var secondsPerSlot = {{ .ChainSecondsPerSlot }}
+    var slotsPerEpoch = {{ .ChainSlotsPerEpoch }}
+    var slotNumber, datepickerUtc, datepickerLocal, beaconchainDay, slotLink
+
+    function slotChanged() {
+      const slot = parseInt(slotNumber.value)
+      if (!isNaN(slot) && slot > 0) {
+        slotLink.href = `/slot/${slot}`
+        const slotDate = new Date((genesisTimeStamp + slot * secondsPerSlot) * 1000)
+        const slotDateLocal = moment(slotDate)
+        const slotDateUtc = moment.utc(slotDate)
+        datepickerUtc.data("daterangepicker").setStartDate(slotDateUtc)
+        datepickerUtc.data("daterangepicker").setEndDate(slotDateUtc)
+        datepickerLocal.data("daterangepicker").setStartDate(slotDateLocal)
+        datepickerLocal.data("daterangepicker").setEndDate(slotDateLocal)
+
+        const timePassed = slotDate.getTime() / 1000 - genesisTimeStamp
+        const daysPassed = Math.max(1, 1 + Math.floor(timePassed / 60 / 60 / 24))
+        beaconchainDay.innerText = daysPassed
```

Comment:
This is tricky: According to the slot finder, epoch 0 is part of beaconchain day 1.
If you check our API though:
* `/api/v1/validator/stats/{index}` returns values for day 0.
* `/api/v1/ethstore/{day}` returns something different for day 0 compared to day 1.

=> Please check if we should display beaconchain 0 for epoch 0.

Reply:
Wrongly started with 1 ... Slots/Epochs/Days start with 0 ... fixed